prbt_hardware_support/test/unit_tests/unittest_brake_test_utils.cpp

```diff
@@ -155,10 +158,16 @@ TEST(BrakeTestUtilsTest, testGetCurrentJointStates)
   /**********
    * Step 2 *
    **********/
-  JointStatesPublisherMock joint_states_pub;
-  auto expected_msg = joint_states_pub.getNextMessage();
+  pilz_testutils::JointStatePublisherMock joint_states_pub;
   joint_states_pub.startAsync();
 
+  // Move away from zero
+  joint_states_pub.setVelocity(0.1);
+  ros::Duration(0.1).sleep();
```

Comment:
sleep during tests makes them flaky. Is there no other way?

Reply:
This is intended, because it will lead to a "random" joint state opposed to just reading zero values.

Reply:
Not really happy about having random behavior inside a unittest. 
If there is no other way at least state as a comment that you want random behavior.